### ISSUE TITLE
[SF][A11Y][Button]: Interactive Controls Must not Be Nested

### ISSUE THREAD

**Issue by sfsf-xweb-sh** [SAP/ui5-webcomponents#4385]
Hi Colleague,

When use ui5-button , set icon="add", using axe to scan page to test a11y issue, the following issue is detected. Please take a look. Thanks!

<img width="1759" alt="Screen Shot 2021-11-24 at 2 24 25 PM" src="https://user-images.githubusercontent.com/78520316/143186219-c47e7fda-9616-439a-b89a-421107a09bf1.png">

UI5 Web Components for React Information:
@ui5/webcomponents version: 0.31.22
@ui5/webcomponents-react version: 0.131.23

**Comment by niyap** [comment]
Hello Colleagues,

UI5 Web Components provide the prerequisites for screen reader support based on the HTML, ARIA, and WCAG standards. All screen readers that follow those standards should work fine. Nevertheless, there are deviations in the interpretation depending on the combination of browser and screen reader. UI5 Web Components focus on compliance with the standards, rely on automated checks for accessibility and manual tests with reference testing environments. 

For Screen Reader Support, we recommend using JAWS 2021 + Chrome (latest), and for HTML/ARIA validation the recommended testing tool is [Access Assistant](https://chrome.google.com/webstore/detail/access-assistant/ojiighldhdmahfdnhfdebnpmlbiemdfm). UI5 Web Components support other environments to the extent of providing a valid HTML and ARIA implementation following the WCAG standards.

Please note that reference testing environments may change over time to reflect changes in the usage of different browsers, their maintenance period, and increased accessibility compliance.

When reporting issues with different testing environments, please ensure that the issue is not false positive, a real accessibility concern is present and there is an impact over the users. Therefore, we recommend to retest the issue using the mentioned reference testing tools and additionally with plain HTML.

In order to process the issues correctly, we would like to have the following information provided:

- Issue description

- Reference to the suspected violated accessibility requirement (e.g. Web Content Accessibility Guidelines, WCAG 2.0, BITV 2.0, EN 301 549)

- Isolated example

- Steps to reproduce

- UI5 Web Components version

- OS/Platform: {...}

- Browser: {...}

- Testing Tool

Thank you in advance!

Kind Regards,
Niya

**Comment by sfsf-xweb-sh** [comment]
Hi @niyap ,

Here is steps to reproduce this issue, I'll send the login info to you on slack.
1. Go to assignment-type page: https://10.236.241.82/sf/manageAssignment?_s.crb=bckHEXE0Lwo2hqRW3o1cKi0gBdTvF2ivEBFJGPzsKiY%253d&#/assignment-type 
2. add chrome extension axe devTools,
3. scan the assignment-type page.

You can get related WCAG requirement by https://dequeuniversity.com/rules/axe/4.3/nested-interactive?application=AxeChrome

Best Regards,
Josephine

**Comment by gmkv** [comment]
Hello @SAP/ui5-webcomponents-topic-b 

There's an ACC issue with the ui5-button as reported by the Axe tool - "Interactive controls must not be nested".

This can also be reproduced in [CodeSandbox here](https://codesandbox.io/s/ui5-webcomponents-forked-986vt?file=/index.html).
Would you have a look?

Regards,
Georgi

**Comment by unazko** [comment]
Hello @SAP/ui5-webcomponents-topic-p,

The described issue reported form the axe tool is caused by a <code><svg tabindex=-1></svg> in the DOM representation of the ui5-icon used inside the ui5-button component.  When the icon is not interactable we already have <code>focusable="false"</code> do we need to supply tabindex well?
Could you please check as we don't have control over this property from the ui5-button component.

Best Regards,
Boyan
